docs/studio/groups.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Groups"
+description: "Manage who can access the resources of your organization."
+icon: circle-info
+sidebarTitle: Overview
+---
+
+Groups allow your organization to have better control over who can access the organization resources by allowing you to customize the rules of what resource each individual role have access to.
```

Comment:
This could be stronger. Groups allow you to manage teams, separate people access for specific resources and much more without managing them manually. Including the benefits of SCIM. A reference to API key should be also good since they work with groups too.